ibis/backends/base/sql/alchemy/translator.py

```diff
@@ -38,7 +38,8 @@ class AlchemyExprTranslator(ExprTranslator):
     def name(self, translated, name, force=True):
         if hasattr(translated, 'label'):
             return translated.label(name)
-        return translated
+        else:
+            return translated
```

Comment:
No real reason to change this line. It's not covered and it's yet another file to review.

Reply:
Removed that branch.